commcare_connect/templates/opportunity/invoice_list.html

```diff
@@ -4,110 +4,31 @@
 {% load i18n %}
 
 {% block top_buttons %}
-    {{ block.super }}
-    {% if not request.is_opportunity_pm %}
-<div x-data="invoiceModal()"
-      @keydown.escape.window="closeModal()"
-      id="invoiceModal"
-    >
-      <button class="button button-md primary-dark" @click="openModal()">
-        {% translate "Create Invoice" %}
-      </button>
+  {{ block.super }}
+  {% if not request.is_opportunity_pm %}
```

Comment:
not sure how much it matters but is the new button behind a switch too? or both workflows will change to use the new button and no longer a modal but only the fields and automated behavior are behind the switch?

Reply:
still curious about this